.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - '[0-9]+.[0-9]+.x'
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: {}
+
+defaults:
+  run:
+    shell: bash
```

Comment:
is this needed? Isn't the default bash all the time?

Reply:
Bash is the default shell in linux, but not in windows.  So while this is just the lint job, eventually when windows is in here too we will need to declare the default shell.

Reply:
fair enough. I don't expect anything to run with Windows in the components repo though